packages/anomaly-detection-store/src/agent/AnomalyDetectionStore.ts

```diff
@@ -0,0 +1,140 @@
+import {IAgentPlugin} from '@veramo/core'
+import {
+  AnomalyDetectionStoreArgs,
+  AnomalyDetectionStoreClearAllLocationsArgs,
+  AnomalyDetectionStoreLocation,
+  AnomalyDetectionStoreLocationPersistArgs,
+  AnomalyDetectionStoreLocationResult,
+  AnomalyDetectionStoreLocationResultIValueData,
+  AnomalyDetectionStoreLocationResultOrUndefined,
+  IAnomalyDetectionStore,
+  schema
+} from '../index'
+
+import {IKeyValueStore, KeyValueStore, ValueStoreType} from "@sphereon/ssi-sdk.kv-store-temp";
+
+export const anomalyDetectionStoreMethods: Array<string> = [
+  'anomalyDetectionStorePersistLocation',
+  'anomalyDetectionStoreHasLocation',
+  'anomalyDetectionStoreRemoveLocation',
+  'anomalyDetectionStoreClearAllLocations',
+  'anomalyDetectionStoreGetLocation',
+  'anomalyDetectionStoreDefaultLocationStore'
+]
+
+/**
+ * {@inheritDoc IAnomalyDetectionStore}
+ */
+export class AnomalyDetectionStore implements IAgentPlugin {
```

Comment:
Please rename to GeoLocationStore. This is only about Geo locations, which happens to be part of anomaly detection. We will not store other data in this plugin in the future. 

So rename the class and also this store plugin. But do not rename the anomaly-detection plugin itself. That just uses the renamed GeoLocationStore module/plugin

Reply:
of course